fiducial_follow/nodes/follow.py

```diff
@@ -178,22 +195,32 @@ def run(self):
                 theta = 0
             # A fiducial was detected since last iteration of this loop
             elif self.got_fid:
+                # ???
                 theta = angular_error * self.angular_rate - theta / 2.0
```

Comment:
It probably would have been better if we'd named the variable `angular_velocity` instead of `theta`
Simple P control would be `angular_velocity = angular_error * self.angular_rate`, however, that tended to overshoot, and subtracting a portion of the previous value was added to provide some damping.  The `2.0` should probably be a parameter.